lib/msf/ui/console/driver.rb

```diff
@@ -406,13 +406,26 @@ def on_startup(opts = {})
 
     run_single("banner") unless opts['DisableBanner']
 
+    av_warning_message if framework.eicar_corrupted?
+
     opts["Plugins"].each do |plug|
       run_single("load '#{plug}'")
     end if opts["Plugins"]
 
     self.on_command_proc = Proc.new { |command| framework.events.on_ui_command(command) }
   end
 
+  def av_warning_message
+      avdwarn = []
+      avdwarn << "Warning: This copy of the Metasploit Framework has been corrupted by an installed anti-virus program."
+      avdwarn << "         We recommend that you disable your anti-virus or exclude your Metasploit installation path,"
+      avdwarn << "         then restore the removed files from quarantine or reinstall the framework. For more info: "
+      avdwarn << "             https://community.rapid7.com/docs/DOC-1273"
+      avdwarn << ""
+
+      avdwarn.map{|line| print_error(line) }
```

Comment:
I would print this to `stderr` to make sure it's not unintentionally ignored or swallowed.

Reply:
Also `80` columns rather than `100` plz. #4776 https://github.com/rapid7/metasploit-framework/pull/14417#issuecomment-731701077